lib/components/STable.vue

```diff
@@ -108,6 +112,10 @@ watch(() => records?.value, () => {
   bodyLock = false
 }, { flush: 'post' })
 
+useResizeObserver(block, ([entry]) => {
+  blockWidth.value = entry.contentRect.width
```

Comment:
Earlier tried using useElementSize/useElementBounding, but those were increasing glitches (delay between head and body) in scrolling.

`blockWidth` can be entirely removed if we can somehow do this from CSS (make the body scroll horizontally). Sadly nothing along the lines of fit-content worked for me.